### PR TITLE
erlang-ls: init at 0.15.0

### PR DESCRIPTION
**Pull request by dlesl**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

## Notes
`erlang-ls` and its dependencies depend on a few rebar3 plugins, but none of them seem to be necessary to build it so I didn't include them.

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 122646` at bbc01c2b run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>erlang-ls</li>
  </ul>
</details>
<details>
  <summary>4 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md">maintainers-missing</a>

Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/beam-modules/erlang-ls/default.nix:28:3:
```
   |
28 |   meta = {
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/development/beam-modules/erlang-ls/default.nix:23:3:
```
   |
23 |   installPhase = ''
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`buildPhase` should probably contain `runHook preBuild` and `runHook postBuild`.

Near pkgs/development/beam-modules/erlang-ls/default.nix:16:3:
```
   |
16 |   buildPhase = ''
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/name-and-version.md">name-and-version</a>

Did you mean to pass `pname` instead of `name` to `mkDerivation`?

Near pkgs/development/beam-modules/erlang-ls/default.nix:9:3:
```
  |
9 |   name = "erlang-ls";
  |   ^
```

Near pkgs/development/beam-modules/erlang-ls/default.nix:8:10:
```
  |
8 |   inherit version;
  |          ^
```
</li>
  </ul>
</details>

**Comment by happysalada** [comment]
I don't have any problems, I'm running the tests on darwin.
I'll wait a bit in case other people want to comment on this.
(In my opinion it's ready to merge).

**Comment by happysalada** [comment]
Result of `nixpkgs-review pr 122646` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>erlang-ls</li>
  </ul>
</details>

File: pkgs/development/beam-modules/erlang-ls/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    mkdir -p $out/bin
+    cp _build/default/bin/erlang_ls $out/bin/
+    cp _build/dap/bin/els_dap $out/bin/
+  '';
+  meta = {
+    homepage = "https://github.com/erlang-ls/erlang_ls";
+    description = "The Erlang Language Server";
+    platforms = lib.platforms.unix;
+    license = lib.licenses.asl20;
+  };
```
Comment (legendofmiracles) [inline pkgs/development/beam-modules/erlang-ls/default.nix:33]:
```suggestion
  meta = with lib; {
    homepage = "https://github.com/erlang-ls/erlang_ls";
    description = "The Erlang Language Server";
    platforms = platforms.unix;
    license = licenses.asl20;
  };
```

**Pull request merged**